### PR TITLE
Add Hagezi's lists

### PR DESCRIPTION
**Pull request by GiddyGoatGaming**

### CODE REVIEW

**Comment by GiddyGoatGaming** [comment]
I really hope i did this right. I was following the instructions by dandelionsprout and collinbarrett but I may have missed something

**Comment by GiddyGoatGaming** [comment]
just realized hagezi's list are already added smh. il be closing this now. sorry

**Comment by collinbarrett** [comment]
> just realized hagezi's list are already added smh. il be closing this now. sorry

Ah! Ok, thanks. FWIW, the PR was pretty good. Thanks for contributing.

**Comment by GiddyGoatGaming** [comment]
> > just realized hagezi's list are already added smh. il be closing this now. sorry
> 
> Ah! Ok, thanks. FWIW, the PR was pretty good. Thanks for contributing.

actually what I could do is add a different formatting of HaGezi's lists, il reopen this and change it to match the rest but for a different format!

**Comment by GiddyGoatGaming** [comment]
Am i missing anything else? I think i did the repositories I'm required

**Review by collinbarrett — CHANGES_REQUESTED** [review]

**Comment by collinbarrett** [comment]
Hmm, the build is complaining about some kind of syntax error in `FilterList.json`. I'm not spotting what it is at the moment.

**Comment by GiddyGoatGaming** [comment]
> Hmm, the build is complaining about some kind of syntax error in `FilterList.json`. I'm not spotting what it is at the moment.

Il check it out:)

**Comment by collinbarrett** [comment]
I'll take another look when I get a minute.

In the meantime, would you mind wiring up some new entries in `FilterListMaintainer.json` (and `Maintainer.json` if needed) as well just to keep these new lists clustered together by maintainer? Thanks.

**Comment by GiddyGoatGaming** [comment]
> I'll take another look when I get a minute.
> 
> In the meantime, would you mind wiring up some new entries in `FilterListMaintainer.json` (and `Maintainer.json` if needed) as well just to keep these new lists clustered together by maintainer? Thanks.

Yep, il do that in a little bit👍

**Comment by GiddyGoatGaming** [comment]
And done:)

File: services/Directory/data/FilterListViewUrl.json [hunk 1]
```diff
@@ -16300,5 +16300,101 @@
...
+    "url": "https://raw.githubusercontent.com/hagezi/dns-blocklists/main/domains/personal.txt"
+  },
+  {
+    "filterListId": 2634,
+    "id": 2900,
+    "primariness": 1,
+    "url": "https://raw.githubusercontent.com/hagezi/dns-blocklists/main/hosts/personal.txt"
+  },
+  {
+    "filterListId": 2635
```
Comment (collinbarrett) [inline services/Directory/data/FilterListViewUrl.json:16395]:
```suggestion
    "filterListId": 2635,
```

**Review by collinbarrett — APPROVED** [review]

**Comment by GiddyGoatGaming** [comment]
Thank you🙏

**Pull request merged**